Funky Friday - New Maps

arcadeboss opened the pull request:
Adding the maps that we are always adding to FF, plus some new ones made for FF.

zzufx commented:
We've been planning to update the FF pool with maps Shimane gathered from the Discord server, as well as some removals - some of the maps you've proposed need updating, and some of the "normal" ones don't really fit the pool and are best added manually in my opinion. "Procedural: " is also duplicated in Shroom Towers

arcadeboss commented:
Yeah, I remember submitting some stuff over 3 months ago, but it didn't seem like it was going anywhere.

Should we update this with your suggestions?  If so, what are the suggestions?  
Or is there another PR/commit ready with other updates?

Fixed incorrect title, and removed Procedural: Cliff Maze, as that is novel, but not funky.

The pool could stand a few slightly normal-ish maps like Emergency Meeting and Lunar Crescent, imo.
